docs/src/main/sphinx/overview/concepts.md

```diff
@@ -34,18 +34,25 @@ servers and other components of Trino's architecture.
 (trino-concept-cluster)=
 ### Cluster
 
-A Trino cluster consists of a {ref}`coordinator <trino-concept-coordinator>` and
-many {ref}`workers <trino-concept-worker>`. Users connect to the coordinator
-with their {ref}`SQL <glossSQL>` query tool. The coordinator collaborates with the
-workers. The coordinator and the workers access the connected
-{ref}`data sources <trino-concept-data-source>`. This access is configured in
-{ref}`catalogs <trino-concept-catalog>`.
+A Trino cluster consists of many Trino servers as [nodes](trino-concept-node)_ -
```

Comment:
Super nitpick but we already say "many" in the next sentence, so it feel repetitive 😅 

This wording also implies that single-node clusters don't exist, when they do. I'm not 100% sure we want to explain this in the glossary as it's a bit of a special case and could be more confusing than helpful.

How about...
```
A Trino cluster consists of several Trino [nodes](trino-concept-node) - one [coordinator](trino-concept-coordinator) and zero-to-many [workers](trino-concept-worker). Users connect....
```

And yeah as mentioned about the helpful-v-confusing part, happy to also replace `zero-to-many` with `many` 👍